lib/DBIx/Class/Helper/ResultSet/IgnoreWantarray.pm

```diff
@@ -5,9 +5,7 @@ package DBIx::Class::Helper::ResultSet::IgnoreWantarray;
 use strict;
 use warnings;
 
-use parent 'DBIx::Class::ResultSet';
-
-sub search :DBIC_method_is_indirect_sugar{
```

Comment:
Why did you remove this attribute?  Is this no longer needed?

Reply:
I grepped for it in the source code of the latest released DBIC, and couldn't find it, so my tests failed. I don't know how they pass for you and CI, I suspect I'm missing something crucial.